.github/workflows/ciV0.yml

```diff
@@ -76,6 +76,10 @@ jobs:
           name: ipyradiant dist ${{ github.run_number }}
           path: ./dist
 
+      -name: run pytests
```

Comment:
add space, e.g. `- name: run pytest`? or just omit `name` altogether (it's short and self-explanatory)